glotaran/model/dataset_model.py

```diff
@@ -41,9 +42,13 @@ def iterate_megacomplexes(
         self,
     ) -> Generator[tuple[Parameter | int | None, Megacomplex | str], None, None]:
         """Iterates of der dataset model's megacomplexes."""
```

Comment:
of der -> over the
-> dataset_models's megacomplexes

Reply:
Fixed in [4c836d6](https://github.com/glotaran/pyglotaran/pull/983/commits/4c836d6b642e40f656da3ba823ffe6bf6625f453)